soh/src/overlays/actors/ovl_En_Wallmas/z_en_wallmas.c

```diff
@@ -327,7 +327,11 @@ void EnWallmas_WaitToDrop(EnWallmas* this, PlayState* play) {
     }
 
     if (this->timer == 0) {
-        EnWallmas_SetupDrop(this, play);
+        if (this->actor.params == WMT_SHADOWTAG && player->stateFlags1) {
```

Comment:
Do we want to compare to specific state flags here? This will return true if any flag in stateFlags1 is set

Reply:
(question from just not knowing)
This is only pointing from the player right?

What else other than cutscenes and text dialogue sets the same stateFlag?

Reply:
https://github.com/HarbourMasters/Shipwright/blob/738172d9edb12a0f2757596ed89ff4737e7032be/soh/include/z64player.h#L421-L452

Reply:
Ah, hidden away in a player file. I can go through the list and indicate the state more specifically.

Reply:
Latest commit should now address this.